### PR TITLE
set only one configuration item once

### PR DESCRIPTION
**Pull request by Xiaochao8**

### CODE REVIEW

File: lib/sgtn-client/util/validate-util.rb [hunk 1]
```diff
@@ -6,31 +6,31 @@ module SgtnClient
...
         end
         
-        disable_cache = Config.configurations[env]["disable_cache"]
+        disable_cache = SgtnClient.config.disable_cache
         if disable_cache != nil && disable_cache != false && disable_cache != true
           messages = messages +  "Configuration[disable_cache] has to be a 'true' or 'false'!\n"
         end
         
-        if messages != "\n"
+        if messages != ""
```
Comment (jessiejuachon) [inline lib/sgtn-client/util/validate-util.rb:33]: Could you change "messages" to something more descriptive like "error_messages"? Also, could you make this a list/array instead of one string and pass to MissingConfig?

This can be in a separate PR if you prefer.

**Review by jessiejuachon — CHANGES_REQUESTED** [review]

File: lib/sgtn-client/util/validate-util.rb (hunk 1 above)
Comment (Xiaochao8) [reply lib/sgtn-client/util/validate-util.rb:33]: Yes. Let's change it in another PR.

**Comment by Xiaochao8** [comment]
> Could you fix the codacy issues and smoke test failures?

codacity issues are fixed.
Devops is fixing an issue running smoke test.

**Review by jessiejuachon — APPROVED** [review]
